_layouts/post.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <title>{{ page.title }}: _{{ site.blog_title | default: site.github.repository_name }}_ by {{ site.owner_name }}</title>
+
+    <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    <meta name="viewport" content="width=device-width">
+    <!--[if lt IE 9]>
+    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <h2><em><a href="../../../../">{{ site.blog_title }}</a></em></h2>
+        <p>a blog</p>
+
+        {% if site.github.is_project_page %}
+          <p class="view"><a href="{{ site.github.repository_url }}">View the Project on GitHub <small>{{ github_name }}</small></a></p>
+        {% endif %}
+
+        {% if site.github.is_user_page %}
+          <p class="view"><a href="{{ site.github.owner_url }}">View My GitHub Profile</a></p>
+        {% endif %}
+
+        {% if site.show_downloads %}
+          <ul>
+            <li><a href="{{ site.github.zip_url }}">Download <strong>ZIP File</strong></a></li>
+            <li><a href="{{ site.github.tar_url }}">Download <strong>TAR Ball</strong></a></li>
+            <li><a href="{{ site.github.repository_url }}">View On <strong>GitHub</strong></a></li>
+          </ul>
+        {% endif %}
+        <h1>{{ page.title }}</h1>
+        
+        <p class="view">by {% if page.author %}{{ page.author }}{% else %}{{site.author}}{% endif %}</p>
+
+        <small>{% if page.tags %}tags: <em>{{ page.tags | join: "</em> - <em>" }}</em> {% endif %}</small>
```

Comment:
It looks like this section, after `show_downloads` is the only change from the default theme? Perhaps then, we could let the `post` layout inherit from the `default` layout to DRY things up (so we're not implementing subsequent changes in two locations)?

To do so, we'd add 

```yaml
---
layout: default
---
```

To the top of this file, and then add the content we want (e.g., the post meta and `{{ content }}` tag), which will take the place of `{{ content }}` in the default layout.

Reply:
I see that I left in an unneeded <br> in l. 48 (which I've now removed), but otherwise there's a change to the title in l. 6, the header in l. 17-18, and the bit you mention. If your suggestion handles that, then it's fine with me. Seems a lot simpler, for sure!